conf/systemd/plackup_ledgersmb.service

```diff
@@ -0,0 +1,33 @@
+# Place this in /etc/systemd/system/plackup_ledgersmb.service; enable & start
```

Comment:
commands how to enable systemd and starting the server sound superfluous, but our users might need that nonetheless.